### ISSUE TITLE
Extra padding on equipment page / drawer covering footer

### ISSUE THREAD

**Issue by shanaqui** [HabitRPG/habitica#11863]
Reported by wilkoklak in Report a Bug:

> Hello, I think I've found a bug. It can be seen in Equipment (and I guess anywhere where a drawer is present) - when the bottom drawer is shown it seems that the main section (or standard-page as I think it's referenced in the source code) of the page has increased padding at the bottom. When I close the drawer, the additional padding is gone. I can see in a comment in the source code that this is to ensure that the drawer doesn't overlap other content, however the drawer isn't even near the standard-page element when windows is scrolled down to the max. I guess that it should add more padding to the bottom of the whole page, because right now the drawer overlaps the footer.
> 
> ![image](https://user-images.githubusercontent.com/8144640/74483884-5aedd200-4eaf-11ea-9a2d-61933f69f09a.png)
> ![image](https://user-images.githubusercontent.com/8144640/74483889-5e815900-4eaf-11ea-820d-c1632b32fa0a.png)

Confirmed in Firefox.

@Tressley, could you confirm how this is supposed to look? (Or tag in someone who can?) :D The person who reported this already volunteered to potentially do a fix.

**Comment by artur-borys** [comment]
Hello, I'm the one who reported this, so you can assign me to this issue :)

**Comment by shanaqui** [comment]
Thanks! But please wait until the designers have confirmed how this is supposed to behave. :)

**Comment by Alys** [comment]
I've added labels, including the "on hold - needs design" one since a design decision is needed.

@artur-borys When a decision is made, there'll be a comment here about it. One of us will then mark this as in progress for you, but please let us know then if you're no longer interested and we'll set it back to help wanted. Thank you! :)

**Comment by Tressley** [comment]
That's quite the jump! I believe this is an artifact from when pages used a smaller footer. Now that we're using a larger footer, the amount of space when the drawer is collapsed is correct.

**Comment by artur-borys** [comment]
@Tressley but the space is added above the footer and I think it should be added below. I'll demonstrate what I have already done in my local branch in a moment

**Comment by artur-borys** [comment]
Okay, here's how it looks when the drawer is open after my changes
![drawer-shown](https://snipboard.io/p94mlW.jpg)

You can see here that when user is at the bottom of the page, there is an extra space below content of the footer, opposed to extra space above footer as it is currently on habitica.com.

When user closes drawer or goes to a page where there is no drawer, the extra space is removed

**Comment by Tressley** [comment]
@artur-borys -- There isn't a reason for padding to be added as the footer allows the drawer to be open without blocking the main content. You can see the correct behavior on the Tasks page.

**Comment by artur-borys** [comment]
@Tressley okay, understood. I'm waiting on the last word about how it should behave and for help wanted label :)

**Comment by Alys** [comment]
@artur-borys I've marked this as in progress for you - thanks! The desired fix is to make the drawer behave as it currently does on the Tasks page, since Tressley has said that's the correct behaviour.

**Comment by artur-borys** [comment]
I don't see any drawer on the Tasks page, so I'm assuming that you just want to remove the extra padding that can be seen in the screenshot at the top of this issue? Sorry for asking, but I just want to make sure that I understood everything.

**Comment by Alys** [comment]
@artur-borys The drawer is visible when you've selected a class and it displays your class skills. It won't be visible for players below level 10 or players who have opted out of the class system.

**Comment by artur-borys** [comment]
@Alys that explains a lot, because I'm <10 lvl. I will check on local instance later.

EDIT: Everything seems to work as expected
